cdn/snippets.py

```diff
@@ -71,6 +71,41 @@ def sign_url(url, key_name, base64_key, expiration_time):
 # [END sign_url]
 
 
+# [START sign_cookie]
```

Comment:
These region tags aren't in the proper format (should be `product_sign_cookie`).

Reply:
Done.